example/usps_api/config/config_parser.h

```diff
@@ -11,11 +11,15 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 // License for the specific language governing permissions and limitations under
 // the License.
+#ifndef CONFIG_PARSER_H
+#define CONFIG_PARSER_H
+
 #include "proto/usps_api/sfc_filter.pb.h"
 #include "json/json.h"
 #include <string>
 #include <list>
 
+
 namespace usps_api_server {
   class Config {
```

Comment:
We usually don't need to indent code inside namespaces, https://google.github.io/styleguide/cppguide.html#Namespace_Formatting.

Reply:
Fixed this for all of the code